zuul.d/projects.yaml

```diff
@@ -198,6 +198,10 @@
     name: github.com/ansible-collections/community.digitalocean
     templates:
       - publish-to-galaxy-3pci
+- project:
```

Comment:
I wonder if you need to rebase this PR?

Reply:
I did, this branch contains the merge commit. Maybe Zuul takes a little bit of time to reload the config and I created this PR too quick... :)

Reply:
There is a promote job that runs, which lays down the updated zuul tenant file, then reload the schedule. Once that is finished, this PR would have worked.